CHANGELOG.md

```diff
@@ -10,114 +10,121 @@ The following classes, methods, properties and events have been deprecated for a
+*    PrintParameters deprecated since version 4.20. Use PrintParameters instead.
+*    PrintTask deprecated since version 4.20. Use print instead.
+*    PrintTemplate deprecated since version 4.20. Use PrintTemplate instead.
+*    projection.isSupported deprecated since version 4.18.
+*    ProjectParameters deprecated since version 4.20. Use ProjectParameters instead.
+*    promiseUtils.reject deprecated since version 4.19. Use the native Promise.reject method instead.
+*    promiseUtils.resolve deprecated since version 4.19. Use the native Promise.resolve method instead.
+*    Query deprecated since version 4.20. Use Query instead.
+*    QueryTask deprecated since version 4.20. Use query instead.
+*    RasterData deprecated since version 4.20. Use RasterData instead.
+*    RelationParameters deprecated since version 4.20. Use RelationParameters instead.
+*    RelationshipQuery deprecated since version 4.20. Use RelationshipQuery instead.
+*    RouteParameters deprecated since version 4.20. Use RouteParameters instead.
+*    RouteResult deprecated since version 4.20. Use RouteResult instead.
+*    RouteTask deprecated since version 4.20. Use route instead.
+*    SceneView.constraints.collision deprecated since version 4.8. Use Ground.navigationConstraint instead.
+*    ServiceAreaParameters deprecated since version 4.20. Use ServiceAreaParameters instead.
+*    ServiceAreaSolveResult deprecated since version 4.20. Use ServiceAreaSolveResult instead.
+*    ServiceAreaTask deprecated since version 4.20. Use serviceArea instead.
+*    SizeVariable.expression deprecated since version 4.2. Use SizeVariable.valueExpression instead.
+*    Slider.labelsVisible deprecated since version 4.15. Use Slider.visibleElements.labels instead.
+*    Slider.rangeLabelsVisible deprecated since version 4.15. Use Slider.visibleElements.rangeLabels instead.
+*    StatisticDefinition deprecated since version 4.20. Use StatisticDefinition instead.
+*    symbolPreview.renderPreviewHTML deprecated since version 4.11. Use symbolUtils.renderPreviewHTML instead.
+*    symbolPreview deprecated since version 4.11. Use symbolUtils instead.
+*    Task deprecated since version 4.20.
+*    TimeSlider.values deprecated since version 4.20. Use timeExtent instead.
+*    TimeSliderViewModel.values deprecated since version 4.20. Use timeExtent instead.
+*    TrimExtendParameters deprecated since version 4.20. Use TrimExtendParameters instead.
+*    widget.renderable deprecated since version 4.19. All properties are automatically tracked now and don't need to be decorated with this decorator.
+*    `SmartMapping.params.basemap` deprecated since version 4.13. Use `view` instead.
+*    The light-blue, dark-blue, light-green, dark-green, light-purple, dark-purple, light-red, dark-red are deprecated since 4.19. Please use `light` or `dark` instead, or create your own theme.
+
 
 BELOW ARE THE MANUALLY ADDED DEPRECATED CLASSES, PROPERTIES, METHODS, EVENTS
```

Comment:
this should not be here

Reply:
it should be one cohesive list, not separated by manual vs automatic

Reply:
Hmmm, okay. I was just copying what we had left over from the 4.21.

Reply:
Done.